### PR TITLE
Add support for Circe in Slick.

### PR DESCRIPTION
**Pull request by Petesta**

### CODE REVIEW

**Comment by Petesta** [comment]
This PR is for [issue 261](https://github.com/tminglei/slick-pg/issues/261).

I will update the `README.md` as well to include information about [circe](https://github.com/travisbrown/circe).

File: addons/circe-json/src/main/scala/com/github/tminglei/slickpg/PgCirceJsonSupport.scala [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    if (driver.isInstanceOf[ExPostgresDriver]) {
+      driver.asInstanceOf[ExPostgresDriver].bindPgTypeToScala("json", classTag[Json])
+      driver.asInstanceOf[ExPostgresDriver].bindPgTypeToScala("jsonb", classTag[Json])
+    }
+
+    implicit class PgJsonPositionResult(r: PositionedResult) {
+      def nextJson() = nextJsonOption().getOrElse(Json.Empty)
+      def nextJsonOption() = r.nextStringOption().map(parse(_).getOrElse(Json.Empty))
+      // def nextJsonOption() = r.nextStringOption().map(parse(_))
```
Comment (Petesta) [inline addons/circe-json/src/main/scala/com/github/tminglei/slickpg/PgCirceJsonSupport.scala]: Will remove.

**Comment by tminglei** [comment]
Merged. _Thanks! :-)_

File: addons/circe-json/src/main/scala/com/github/tminglei/slickpg/PgCirceJsonSupport.scala (hunk 1 above)
Comment (tminglei) [inline addons/circe-json/src/main/scala/com/github/tminglei/slickpg/PgCirceJsonSupport.scala]: I removed it after merged.

**Pull request merged**